DQMOffline/Hcal/interface/HcalRecHitsDQMClient.h

```diff
@@ -47,13 +59,19 @@ class HcalRecHitsDQMClient : public DQMEDHarvester {
   std::string dirNameJet_;
   std::string dirNameMET_;
 
+  edm::ESHandle<CaloGeometry> geometry ;
```

Comment:
is this used outside of the function that defines it ? If not, no need for it to be a member data